ipinfo/config.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/ipinfo/cli/lib"
+)
+
+type Config struct {
+	GlobalCache bool
+}
+
+// init function
+func InitConfig() error {
+	path, err := ConfigPath()
+	if err != nil {
+		return err
+	}
+	if !lib.FileExists(path) {
+		gConfig.GlobalCache = true
+		SetConfig(gConfig)
+	}
+	return nil
+}
+
+// returns the path to config file.
+func ConfigPath() (string, error) {
+	confDir, err := getConfigDir()
```

Comment:
I think we can probably move the `getConfigDir` func in `utils_config` into this new file now, since it's all strongly related.